opentelekomcloud/resource_opentelekomcloud_rds_instance_v1.go

```diff
@@ -0,0 +1,618 @@
+}
+
+func resourceInstanceRead(d *schema.ResourceData, meta interface{}) error {
+	config := meta.(*Config)
+	client, err := config.RdsV1Client(GetRegion(d, config))
+	if err != nil {
+		return fmt.Errorf("Error creating OpenTelekomCloud rds client: %s", err)
+	}
+
+	instanceID := d.Id()
+	instance, err := instances.Get(client, instanceID).Extract()
+	if err != nil {
+		return CheckDeleted(d, err, "instance")
+	}
+
+	log.Printf("[DEBUG] Retrieved instance %s: %#v", instanceID, instance)
+
+	if instance.Name != "" {
+		nameList := strings.Split(instance.Name, "-"+instance.DataStore.Type)
+		log.Printf("[DEBUG] Retrieved nameList %#v", nameList)
+		if len(nameList) > 0 {
+			d.Set("name", nameList[0])
+		}
+	}
+
+	d.Set("hostname", instance.HostName)
+	d.Set("type", instance.Type)
+	d.Set("region", instance.Region)
+	d.Set("availabilityzone", instance.AvailabilityZone)
+	d.Set("vpc", instance.Vpc)
+	d.Set("status", instance.Status)
+
+	nicsList := make([]map[string]interface{}, 0, 1)
+	nics := map[string]interface{}{
+		"subnetid": instance.Nics.SubnetId,
+	}
+	nicsList = append(nicsList, nics)
+	log.Printf("[DEBUG] nicsList: %+v", nicsList)
+	if err := d.Set("nics", nicsList); err != nil {
+		return fmt.Errorf("[DEBUG] Error saving nics to Rds instance (%s): %s", d.Id(), err)
+	}
+
+	securitygroupList := make([]map[string]interface{}, 0, 1)
+	securitygroup := map[string]interface{}{
+		"id": instance.SecurityGroup.Id,
+	}
+	securitygroupList = append(securitygroupList, securitygroup)
+	log.Printf("[DEBUG] securitygroupList: %+v", securitygroupList)
+	if err := d.Set("securitygroup", securitygroupList); err != nil {
+		return fmt.Errorf("[DEBUG] Error saving securitygroup to Rds instance (%s): %s", d.Id(), err)
+	}
+
+	d.Set("flavorref", instance.Flavor.Id)
+
+	volumeList := make([]map[string]interface{}, 0, 1)
+	volume := map[string]interface{}{
+		"type": instance.Volume.Type,
+		"size": instance.Volume.Size,
+	}
+	volumeList = append(volumeList, volume)
+	if err := d.Set("volume", volumeList); err != nil {
+		return fmt.Errorf(
+			"[DEBUG] Error saving volume to Rds instance (%s): %s", d.Id(), err)
+	}
+
+	d.Set("dbport", instance.DbPort)
+
+	datastoreList := make([]map[string]interface{}, 0, 1)
+	datastore := map[string]interface{}{
+		"type":    instance.DataStore.Type,
+		"version": instance.DataStore.Version,
+	}
+	datastoreList = append(datastoreList, datastore)
+	if err := d.Set("datastore", datastoreList); err != nil {
+		return fmt.Errorf(
+			"[DEBUG] Error saving datastore to Rds instance (%s): %s", d.Id(), err)
+	}
+
+	//haList := make([]map[string]interface{}, 0, 1)
+	//ha := map[string]interface{}{
+	//	"enable":          instance.Ha.Enable,
+	//	"replicationmode": instance.Ha.ReplicationMode,
+	//}
+	//haList = append(haList, ha)
+	//if err := d.Set("ha", haList); err != nil {
+	//	return fmt.Errorf(
+	//		"[DEBUG] Error saving haSet to Rds instance (%s): %s", d.Id(), err)
+	//}
+
+	d.Set("updated", instance.Updated)
```

Comment:
We are trying to standardize date/time usage to the **RFC-3339** format which go supports in their [`time`](https://golang.org/pkg/time/) package. It would be nice to parse these from the API format and set them to RFC-3339 if possible, but not a requirement.